lib/season_stats_module.rb

```diff
@@ -60,7 +60,7 @@ def most_accurate_team(season_id)
       team_hash[team] = (total_goals.to_f / total_shots.to_f)
     end
 
-    accurate_team = team_hash.max_by do |team, shot_percentage|
+    accurate_team = team_hash.max_by do |_team, shot_percentage|
```

Comment:
Is the underscore what rubocop suggests for variables that shadow other variables?